pkg/plugins/runtime/gateway/route/configurers.go

```diff
@@ -0,0 +1,378 @@
+package route
+
+import (
+	"net/http"
+
+	envoy_config_core_v3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
```

Comment:
I think we usually import this as `envoy_core`, `envoy_route` and `envoy_type_matcher`. Do you want to deliberately use "v3" suffix?

Reply:
This is the import alias that Goland generates. I'll switch to `envoy_core` and add that to the golint config.

Reply:
```$ git grep envoy/config/core/v3 | awk '$1~/.*.go/{print $2}' | sort | uniq -c
      1 -
      1 _
     11 envoy_config_core_v3
     53 envoy_core
      1 envoy_core_v3
```

I'll update these ones, but do the lint changes in a separate PR.

Reply:
I ended up using `envoy_config_core` and `envoy_config_route` as the alias names because I am already importing Kuma envoy wrappers with the alias `envoy_routes` (which is also a standard alias. Using both `envoy_route` and `envoy_routes` as aliases in the same file would be confusing :)